lib/api/model/model.dart

```diff
@@ -766,77 +839,38 @@ class StreamMessage extends Message {
 }
 
 @JsonSerializable(fieldRename: FieldRename.snake)
-class DmRecipient {
```

Comment:
nit:

    This is unused in app code.  This is mostly NFC except that we will
    ignore fields other than "id" from the list of objects from
    "display_recipient" on message events.

s/message events/message objects from the server/

These objects also notably appear in getMessages responses.